### DISCUSSION TITLE
how to install wimlib

### DISCUSSION THREAD

**Discussion by Cocobeck** [Homebrew/discussions#4055] [Getting started]
Hello, i don't really enderstand Homebrew,but i need to use it to do something and i need to instal wimlib with homebrew but when I enter the command brew install wimlib . It's says me some error and says me that : ==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
Is something wrong with my command or i need to instal something more?

**Comment by SMillerDev** [comment]
What does `brew doctor` say?

**Reply by Cocobeck** [reply]
it's says me that 
Warning: The following directories are not writable by your user:
/usr/local/Cellar

You should change the ownership of these directories to your user.
  sudo chown -R $(whoami) /usr/local/Cellar

And make sure that your user has write permission.
  chmod u+w /usr/local/Cellar

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 13.2.1.


Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout master
And after that i did a update of Xcode and command line tool like it says

**Reply by gromgit** [reply]
>```
>Warning: Homebrew/homebrew-core was not tapped properly! Run:
>rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
>brew tap homebrew/core
>```

Here's your problem...and the solution. The other warnings also looks like they need addressing.

**Reply by Cocobeck** [reply]
Yes it's work thank you
